db/__init__.py

```diff
@@ -821,6 +823,23 @@ def __repr__(self):
         )
 
 
+class SolendHealthRatio(Base):
+    __tablename__ = "solend_health_ratios"
+    __table_args__ = {"schema": SCHEMA_LENDERS}
+
+    slot = Column(BigInteger, primary_key=True, nullable=False)
+    last_update = Column(BigInteger, index=True, nullable=True)
+    user = Column(String, primary_key=True, nullable=False)
+    health_factor = Column(String, index=True, nullable=False)
+    std_health_factor = Column(String, index=True, nullable=False)
+    collateral = Column(String, nullable=False)
+    risk_adjusted_collateral = Column(String, nullable=False)
+    debt = Column(String, nullable=False)
+    risk_adjusted_debt = Column(String, nullable=False)
+    protocol = Column(String, default='solend', nullable=False)
+    timestamp = Column(DateTime, default=datetime.utcnow, nullable=False)
```

Comment:
Wouldn't it be better/mote consistent if this was a unix timestamp?

Reply:
agree.  fixed